src/welcomePage.ts

```diff
@@ -4,8 +4,13 @@
 "use strict";
 import * as fs from "fs";
 import * as path from "path";
+import * as semver from "semver";
 import * as vscode from "vscode";
 import { Constants } from "./constants";
+import { TelemetryClient } from "./telemetryClient";
+
+const packageJSON = vscode.extensions.getExtension(Constants.ExtensionId).packageJSON;
+const extensionVersion: string = packageJSON.version;
```

Comment:
Looks like the above two variables can be removed to method scope instead of global scope.

Reply:
fixed